.github/actions/eks-manage-cluster/README.md

```diff
@@ -0,0 +1,76 @@
+# Deploy or Destroy EKS Cluster
+
+This GitHub Action automates the deployment or destruction of an Amazon Elastic Kubernetes Service (EKS) cluster using Terraform. It also installs necessary tools like Terraform, AWS CLI, and `kubectl`, and sets up the Kubernetes context for the created cluster.
+
+## Usage
+
+To use this action, add it to your workflow file (e.g., `.github/workflows/eks-deploy.yml`):
+
+```yaml
+name: EKS Cluster Management
+
+on:
+  workflow_dispatch:
+
+jobs:
+  eks_management:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Deploy or Destroy EKS Cluster
+        uses: camunda/camunda-tf-eks-module/eks-manage-cluster@main
+        with:
+          action: 'create' # or 'destroy'
+          aws-region: 'us-west-2'
+          cluster-name: 'my-eks-cluster'
+          kubernetes-version: '1.30'
+          cluster-service-ipv4-cidr: '10.190.0.0/16'
+          cluster-node-ipv4-cidr: '10.192.0.0/16'
+          np-instance-types: '["t2.medium"]'
+          np-capacity-type: 'SPOT'
+          np-node-desired-count: '4'
+          np-node-min-count: '1'
+          np-disk-size: '20'
+          np-ami-type: 'AL2_x86_64'
+          np-node-max-count: '10'
+          s3-backend-bucket: 'your-terraform-state-bucket'
+          s3-bucket-region: 'us-west-2'
+          tf-modules-revision: 'main'
+          tf-modules-path: './.action-tf-modules/eks/'
+          login: 'true'
+          awscli-version: '2.15.52'
+```
+
+## Inputs
```

Comment:
![](https://media1.tenor.com/m/zAe6N7L7biwAAAAC/tell-us-more-willy-wonka.gif)

same as all the other ones ... required inputs with defaults